docs/hooks-new-language.md

```diff
@@ -17,57 +17,59 @@ Dredd hooks language abstraction bridge comes with [the language agnostic test s
 If you want to write a hook handler for your language you will have to implement:
 
 - CLI Command runnning TCP socket server
-  - [Must return message `Starting` to stdout](https://github.com/apiaryio/dredd-hooks-template/blob/master/features/tcp_server.feature#L5)
-
+    - [Must return message `Starting` to stdout](https://github.com/apiaryio/dredd-hooks-template/blob/master/features/tcp_server.feature#L5)
```

Comment:
The indentation of the nested lists was not correct. It was just 2 spaces and it did not render correctly. Changing to 4 spaces helped.